### PR TITLE
openssh: new article

### PR DESCRIPTION
**Pull request by mcpcpc**

### CODE REVIEW

**Comment by dylanaraps** [comment]
Should this go under `xorg/`?

File: xorg/openssh.txt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+* If you are looking to forward GUI-based applications through an SSH tunnel,
+  refer to the @/x11-forwarding article.
+
+
+References
+________________________________________________________________________________
+
+[0] https://en.wikipedia.org/wiki/OpenSSH
+
```
Comment (dylanaraps) [inline xorg/openssh.txt:86]: Drop newline between references.

File: xorg/openssh.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+IP address of the SSH remote server. Upon pressing return, you will also be
+prompted to enter the password of the regular user specified.
+
+Verify that the key was copied to the remote server and passwordless login works
+by entering the following command from the previous section:
+
+$ ssh USERNAME@SERVER 
+
+Once passwordless login has been verified, disable password authentication on
+the remoter server:
```
Comment (dylanaraps) [inline xorg/openssh.txt:62]: `remoter` -> `remote`

**Comment by mcpcpc** [comment]
> Should this go under `xorg/`?

Struggled with this one... proposed folders/categories:
- [ ] connectivity
- [ ] servers
- [ ] ssh

**Comment by dylanaraps** [comment]
Network is a good option. We can also put the WiFi related pages in this category. It might make even more sense to create a generic "software" category (to include anything that does not fit).

**Comment by mcpcpc** [comment]
> Network is a good option. We can also put the WiFi related pages in this category. It might make even more sense to create a generic "software" category (to include anything that does not fit).

i like the idea of "software".  otherwise it's a rabbit hole of folders and things "just" not fitting.  I'll move it and move wpa_supplicant in a later PR as well.

**Comment by mcpcpc** [comment]
sorry for all the commits.. should be good now.

**Comment by dylanaraps** [comment]
One space after each sentence! Apologies for badgering you with this.

**Comment by mcpcpc** [comment]
> One space after each sentence! Apologies for badgering you with this.

This is my own fault for not being able to write English properly. lol. 

(will have to investigate whether vim spell check can catch this for me in the future)

Corrected.

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**